Can't package beats: error while generating package.yml

When running make package, the process always fails with message
`Failed to parse standard input: yaml: line 10: did not find expected key`
`make[1]: *** [../libbeat/scripts/Makefile:404: prepare-package-cgo] Error 1`
`make[1]: Leaving directory '/home/amarquis/code/go/beats/metricbeat'`
`make: *** [../libbeat/scripts/Makefile:456: package] Error 2`

Seems like the call of sed (dev-tools/packer/xgo-scripts/before_build.sh:58) is the reason for this issue, cause it alters the line 
`:doc_branch:  "master"`
to
`:doc_branch "m" aster`
of the package.yml in {BEAT_PATH}/build

Should be (?)
`:doc_branch: "m"`

--- a/dev-tools/packer/xgo-scripts/before_build.sh
+++ b/dev-tools/packer/xgo-scripts/before_build.sh
@@ -55,7 +55,7 @@ cat ${ES_BEATS}/libbeat/docs/version.asciidoc >> ${PREFIX}/package.yml
 
 # Make variable naming of doc-branch compatible with gotpl. Generate and copy README.md into homedir
 # Add " to the version as gotpl interprets 6.0 as 6
-sed -i -e 's/:doc-branch: \([0-9]*.[0-9]*\)/doc_branch: "\1" /g' ${PREFIX}/package.yml
+sed -i -e 's/:doc-branch: \(.*\)/doc_branch: "\1" /g' ${PREFIX}/package.yml
 
 # Create README file
 /go/bin/gotpl /templates/readme.md.j2 < ${PREFIX}/package.yml > ${PREFIX}/homedir/README.md

